panel-app/src/components/file-explorer/file-explorer.css

```diff
@@ -0,0 +1,108 @@
+#file-explorer .header-section {
```

Comment:
As SCSS, it would be: 

```scss
#file-explorer .header-section {
	padding: 24px 32px;
	border-bottom: 1px solid #e5e8e8;
	height: 110px;
}

.header_title-text {
	color: #282829;
	font-family: Gibson;
	font-size: 18px;
	font-style: normal;
	font-weight: 500;
	line-height: 24px;
	margin-bottom: 4px;
	display: flex;
	align-items: center;
}

#infoToken-img {
	margin-left: 8px;
	cursor: pointer;

	&::part(image) {
		width: 16px;
		height: 16px;
	}
}

.header_sub-text {
	color: #565b66;
	font-family: Gibson;
	font-size: 14px;
	font-style: normal;
	font-weight: 300;
	line-height: 20px; /* 142.857% */
	letter-spacing: 0.042px;
}

.header_file-picker {
	.select-fill-outline {
		border-radius: 8px;
		--border-color: #e5e8e8;
		width: 315px;
		min-height: 35px !important;
		height: 35px;
		margin-top: 16px;
	}

	ion-select::part(container) {
		width: 300px;
	}

	ion-select::part(placeholder) {
		color: #6c727b;
		font-family: Gibson;
		font-size: 14px;
		font-style: normal;
		font-weight: 300;
		line-height: 21px; /* 150% */
		letter-spacing: 0.042px;
	}
}

ion-select.never-flip::part(icon) {
	transform: none;
}

#file-explorer .content-section {
	background-color: #fcfcfc;
	padding: 24px 32px;
  
	.no-file-container {
		display: flex;
		padding: 24px;
		border-radius: 8px;
		border: 1px solid #e5e8e8;
		background: #fff;
		box-shadow: 0px 4px 8px 0px rgba(4, 8, 31, 0.08), 0px 0px 0px 1px #e5e8e8 inset;
		max-width: 500px;
		height: 285px;
		justify-content: space-around;
		align-items: center;

		.info-text {
			margin-bottom: 32px;
			color: #6c727b;
			font-family: Gibson;
			font-size: 18px;
			font-style: normal;
			font-weight: 500;
			line-height: 24px;
		}

		.create-file-btn {
			.button-native {
				border-radius: 8px;
				background: #1372ec;
				width: 140px;
				height: 32px;
				font-family: Gibson;
				font-size: 14px;
				font-style: normal;
				font-weight: 300;
				line-height: 21px;
				letter-spacing: 0.042px;
			}

			&::part(native) {
				text-transform: none;
			}
		}
	}
}
```